docs/docs/refreshing-content.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Refreshing Content
+---
+
+During local development it can be useful to refresh sourced content without restarting the development server. To faciliate this Gatsby exposes an environmental variable `ENABLE_GATSBY_REFRESH_ENDPOINT`.
+
+If set to true, this will expose a /\_\_refresh webhook that is able to receive POST requests to refresh the sourced content. This exposed webhook can be triggered whenever remote data changes.
+
+You can trigger this endpoint locally, for example, on Unix-based operating systems (like Ubuntu and macOS) using curl -X POST http://localhost:8000/__refresh.
+
+Additionally, the sourced content can also be refreshed through the GraphiQL explorer.
```

Comment:
```suggestion
Additionally, the sourced content can also be refreshed through the [GraphiQL explorer](/docs/running-queries-with-graphiql).
```

This is more of a suggestion than correction! I think adding this link might be helpful but I'm happy to leave it to your discretion.

Reply:
In addition, it's not clear from this sentence that there's now a button people can click in order to refresh. Could you get a little more specific here? Maybe something like...

```suggestion
Additionally, the sourced content can also be refreshed with the "Refresh Data" button in the [GraphiQL explorer](/docs/running-queries-with-graphiql). This button is only visible if `ENABLE_GATSBY_REFRESH_ENDPOINT` is set to `true`.
```